document that `torch.triangular_solve` has optional out= parameter

dagitses opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #63253

Fixes #57955

Differential Revision: [D30312134](https://our.internmc.facebook.com/intern/diff/D30312134)

Commit:
document that `torch.triangular_solve` has optional out= parameter

[ghstack-poisoned]

Commit:
Update on "document that `torch.triangular_solve` has optional out= parameter"


Fixes #57955

[ghstack-poisoned]

dagitses commented:
@dagitses has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30312134).

Commit:
Update on "document that `torch.triangular_solve` has optional out= parameter"


Fixes #57955

Differential Revision: [D30312134](https://our.internmc.facebook.com/intern/diff/D30312134)

[ghstack-poisoned]

dagitses commented:
@dagitses has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30312134).

mruberry commented on torch/_torch_docs.py:
```diff
@@ -9739,6 +9739,7 @@ def merge_dicts(*dicts):
     unitriangular (bool, optional): whether :math:`A` is unit triangular.
         If True, the diagonal elements of :math:`A` are assumed to be
         1 and not referenced from :math:`A`. Default: ``False``.
+    out ((Tensor, Tensor), optional): the output tensors
```
To properly document the out parameter it should also appear in the signature.

"out" is a Keyword arg, which we typically separate from the "Args"

dagitses replied:
thanks, i should have looked at some other examples first!

Commit:
Update on "document that `torch.triangular_solve` has optional out= parameter"


Fixes #57955

Differential Revision: [D30312134](https://our.internmc.facebook.com/intern/diff/D30312134)

[ghstack-poisoned]

Commit:
Update on "document that `torch.triangular_solve` has optional out= parameter"


Fixes #57955

Differential Revision: [D30312134](https://our.internmc.facebook.com/intern/diff/D30312134)

[ghstack-poisoned]

dagitses commented:
@dagitses has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30312134).

Commit:
Update on "document that `torch.triangular_solve` has optional out= parameter"


Fixes #57955

Differential Revision: [D30312134](https://our.internmc.facebook.com/intern/diff/D30312134)

[ghstack-poisoned]

dagitses commented:
@dagitses has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30312134).

Commit:
Update on "document that `torch.triangular_solve` has optional out= parameter"


Fixes #57955

Differential Revision: [D30312134](https://our.internmc.facebook.com/intern/diff/D30312134)

[ghstack-poisoned]

dagitses commented:
@dagitses has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30312134).

mruberry reviewed (approved):
Just two minor suggestions inline for your review

Pre-approving for code velocity

Commit:
Update on "document that `torch.triangular_solve` has optional out= parameter"


Fixes #57955

Differential Revision: [D30312134](https://our.internmc.facebook.com/intern/diff/D30312134)

[ghstack-poisoned]

Commit:
Update on "document that `torch.triangular_solve` has optional out= parameter"


Fixes #57955

Differential Revision: [D30312134](https://our.internmc.facebook.com/intern/diff/D30312134)

[ghstack-poisoned]

Commit:
Update on "document that `torch.triangular_solve` has optional out= parameter"


Fixes #57955

Differential Revision: [D30312134](https://our.internmc.facebook.com/intern/diff/D30312134)

[ghstack-poisoned]
